tests/tests_plotting/test_porkchop.py

```diff
@@ -12,8 +12,7 @@ def test_porkchop_plotting():
 
     launch_span = time_range("2005-04-30", end="2005-10-07")
     arrival_span = time_range("2005-11-16", end="2006-12-21")
-    dv_dpt, dv_arr, c3dpt, c3arr, tof = porkchop(
-        Earth, Mars, launch_span, arrival_span, ax=ax
-    )
+    porkchop_plot = PorkchopPlotter(Earth, Mars, launch_span, arrival_span, ax=ax)
```

Comment:
@jorgepiloto do you agree with this API?

Reply:
Since this is a refactor, I think we can merge after requested changes and once all tests pass. 

It would be interesting to add more methods and features, but those can be discussed in associated issue. I will add more comments there :+1: